CONTRIBUTING.md

```diff
@@ -51,3 +52,49 @@ A couple of notes:
 
 - Using a naming convention will allow us to verify that APIs marked `testsOnly` are not being used inside the SDK; we’ll do this in #70.
 - I believe that we should be able to eliminate the boilerplate of re-exposing a `private` member as a `testsOnly` member (as exemplified above) using a macro (called e.g. `@ExposedToTests`), but my level of experience with macros is insufficient to be confident about being able to do this quickly, so have deferred it to #71.
+
+#### Attributing tests to a spec point
+
+When writing a test that relates to a spec point from the Chat SDK features spec, add a comment that contains one of the following tags:
+
+- `@spec <spec-item-id>` — The test case directly tests all the functionality documented in the spec item.
+- `@specOneOf(m/n) <spec-item-id>` — The test case is the m<sup>th</sup> of n test cases which, together, test all the functionality documented in the spec item.
+- `@specPartial <spec-item-id>` — The test case tests some, but not all, of the functionality documented in the spec item. This is different to `@specOneOf` in that it implies that the test suite does not fully test this spec item.
+
+The `<spec-item-id>` parameter should be a spec item identifier such as `CHA-RL3g`.
```

Comment:
**suggestion (documentation):** Standardize terminology: 'spec point' vs 'spec item'

The document uses both 'spec point' and 'spec item'. Consider standardizing on one term for consistency throughout the document.